app/models/inbox_comment.rb

```diff
@@ -16,17 +16,21 @@ class InboxComment < ApplicationRecord
       when 'true' then true
       when 'false' then false
       else [true, false]
-    end
+                 end
+    direction = case filters[:date]&.upcase
```

Comment:
This only has two possible outcomes, so using a `case` statement is overkill. It'd be cleaner to write it as
```
direction = (filters[:date]&.upcase == "ASC" ? "created_at ASC" : "created_at DESC")
```
Or, since `created_at` is shared by both, you could even shorten it to:
```
direction = (filters[:date]&.upcase == "ASC" ? "ASC" : "DESC")
```
And then call `order(created_at: direction)` below.